client/ayon_core/hosts/nuke/api/utils.py

```diff
@@ -1,11 +1,17 @@
 import os
 import re
+import traceback
 
 import nuke
 
-from ayon_core import resources
+from pyblish import util, api
```

Comment:
Please use full import of `pyblish.api` and `pyblish.util`.

Reply:
Done.